polus-feature-extraction-plugin/plugin.json

```diff
@@ -125,6 +131,11 @@
         "title": "Intensity image collection",
         "description": "Intensity image collection"
       },
+      {
+        "key": "inputs.filePattern",
+        "title": "File pattern",
+        "description": "To match intensity and labeled/segmented images"
+      },
```

Comment:
We should hide unitLength and pixelsPerunit if embeddedpixelsize is selected. Also, embeddedpixelsize should be selected by default.